Content.Server/GameObjects/Components/Mobs/BuckleComponent.cs

```diff
@@ -237,11 +237,6 @@ public bool TryUnbuckle(IEntity user, bool force = false)
                 StandingStateHelper.Standing(Owner);
             }
 
-            if (Owner.TryGetComponent(out SpeciesComponent species))
-            {
-                species.CurrentDamageState.EnterState(Owner);
-            }
-
             BuckleStatus();
```

Comment:
So I guess due to removing this, when you unbuckle a dead mob they won't fall to the ground?